install/syndesis.yml

```diff
@@ -156,6 +156,11 @@ parameters:
   generate: expression
   name: CLIENT_STATE_ENCRYPTION_KEY
   required: true
+- description: Volume space available for the upgrade process (backup data), e.g. 512Mi, 2Gi.
```

Comment:
@nicolaferraro sorry for bugging you again, but I see the template changes only in syndesis.yml and syndesis-dev.yml but not in generator/07-syndesis-upgrade.yml.mustache which means that they will be overwritten after the next recreation of the templates. Could you please verify ?

Reply:
Also, there's a pre-commit hook to keep the resulting YAMLs in sync with the templates. hint hint.